src/tags/control/Label.js

```diff
@@ -236,12 +236,18 @@ const Model = types.model({
   },
 
   onHotKey() {
-    if(self.annotation.isDrawing) {
-      return false;
-    } else {
-      self.annotation.unselectAreas();
-      return self.toggleSelected();
-    }
+    return self.onLabelInteract();
+  },
+
+  onClick() {
+    return self.onLabelInteract();
+  },
+
+  onLabelInteract() {
+    if(!self.annotation.isDrawing) {
+      self.toggleSelected();
```

Comment:
what is the return of `toggleSelected`? I seen this was previously returning the result of that function in `onHotKey`. Just want to make sure this is returning the equivalence as previous in all scenarios.

Reply:
appears to return nothing based on what I see here https://github.com/heartexlabs/label-studio-frontend/blob/0aafae54f03808dddbce54642c1aac281ea487d4/src/tags/control/Label.js#L114

Reply:
I opted to return false as the onClick was returning false